livekit-rtc/livekit/rtc/room.py

```diff
@@ -55,7 +59,7 @@ def __init__(self, message: str):
         self.message = message
 
 
-class Room(EventEmitter):
+class Room(EventEmitter[EventTypes]):
```

Comment:
So in this PR we still won't get typing for the Callback args right?

Reply:
No, I didn't find a good solution to do it, so atm there is only auto_complete for the type of event